commands/apps.go

```diff
@@ -307,19 +308,48 @@ func RunAppsUpdate(c *CmdConfig) error {
 		return err
 	}
 
-	app, err := c.Apps().Update(id, &godo.AppUpdateRequest{Spec: appSpec})
+	create, err := c.Doit.GetBool(c.NS, "create")
 	if err != nil {
 		return err
 	}
 
+	exists := true
+	if create {
+		_, err := c.Apps().Get(id)
+		if strings.Contains(err.Error(), "400") {
+			exists = false
+		}
+	}
+
+	var app *godo.App
+	if exists {
+		app, err = c.Apps().Update(id, &godo.AppUpdateRequest{Spec: appSpec})
+		if err != nil {
+			return err
+		}
+	} else {
+		notice("App %s was not existing, creating new app", id)
+
+		app, err = c.Apps().Create(&godo.AppCreateRequest{Spec: appSpec})
+		if err != nil {
+			return err
+		}
+	}
+
 	wait, err := c.Doit.GetBool(c.NS, doctl.ArgCommandWait)
 	if err != nil {
 		return err
 	}
 
 	if wait {
 		apps := c.Apps()
-		notice("App update is in progress, waiting for app to be running")
+
+		if exists {
```

Comment:
Also, I am not sure about how notice should be handled in different scenarios. This is probably the best way.